guides/gatherers.md

```diff
@@ -14,32 +14,49 @@ Facts Gathering process in a nutshell
 fact = gatherer(argument)
 ```
 
+## A note of gatherers versioning
+
+When a gatherer includes a breaking change (e.g. the Rhai output format changes), a new version of that same gatherer will be introduced with a new
```

Comment:
`When a change on a gatherer includes...`
And avoid future term `... of that same gatherer is introduced with a new...`